inc/woocommerce/class-storefront-woocommerce.php

```diff
@@ -150,11 +150,6 @@ public function woocommerce_scripts() {
 			if ( ! class_exists( 'Storefront_Sticky_Add_to_Cart' ) && is_product() ) {
 				wp_register_script( 'storefront-sticky-add-to-cart', get_template_directory_uri() . '/assets/js/sticky-add-to-cart' . $suffix . '.js', array(), $storefront_version, true );
 			}
-
-			if ( defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '3.3', '<' ) ) {
```

Comment:
We should probably search the code for other uses of `WC_VERSION` - maybe there's other stuff we can tidy up.

Reply:
Sure thing! I found this one while working on #1369, but there might be others.

Reply:
#1373 💥